commit bd07cc3706044146df33b3a6518396a8e1efda44 (refs/changes/40/3440/1)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2018-02-22T16:32:24+00:00 (1 year, 7 months ago)
    
    OS-6659 sdev_plugin_register should set *errp to 0 on success

diff --git a/usr/src/uts/common/fs/dev/sdev_plugin.c b/usr/src/uts/common/fs/dev/sdev_plugin.c
index 4f862ccb72..a84aec4c59 100644
--- a/usr/src/uts/common/fs/dev/sdev_plugin.c
+++ b/usr/src/uts/common/fs/dev/sdev_plugin.c
@@ -661,6 +661,8 @@ sdev_plugin_register(const char *name, sdev_plugin_ops_t *ops, int *errp)
 	rw_exit(&sdp->sdev_contents);
 	VN_RELE(vp);
 
+	*errp = 0;
+
 	return ((sdev_plugin_hdl_t)spp);
 }
 
